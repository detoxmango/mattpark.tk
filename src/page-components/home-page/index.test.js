import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'

import HomePage from '.'

describe('HomePage', () => {
    const renderer = ShallowRenderer.createRenderer()
    let tree

    beforeEach(() => {
        renderer.render(<HomePage />)
        tree = renderer.getRenderOutput()
    })

    it('renders correctly', () => {
        expect(tree).toMatchSnapshot()
    })

    it('passes', () => {
        expect(true).toBe(true)
    })
})
