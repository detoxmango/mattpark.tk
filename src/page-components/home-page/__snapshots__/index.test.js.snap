// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
<Unknown
  classes={
    Object {
      "letter": "Component-letter-3",
      "letterText": "Component-letterText-4",
      "miniStripe": "Component-miniStripe-1",
      "stripe": "Component-stripe-2",
      "svg": "Component-svg-5",
    }
  }
/>
`;
